### PR TITLE
CardBody: change default value of `isScrollable` to `false`

### PR DESCRIPTION
**Pull request by ciampo**
## Description

Change the default value of the `isScrollable` prop in the `<CardBody />` component from `true` to `false`
This fix is one way to solve the bug highlighted in https://github.com/WordPress/gutenberg/issues/33439, although it has the potential to introduce breaking changes too.

## How has this been tested?

- Unit test pass
- Check the stories using `<CardBody />` in Storybook:
  - The Card story
  - The Flyout story
  - The ContextSystemProvider story

## Screenshots 

N/A

## Types of changes

Fix + potential minor breaking change

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] I've tested my changes with keyboard and screen readers. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- N/A I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal).

### CODE REVIEW

**Comment by ciampo** [comment]
Looking at the changelog, the recent changes in the `Card` component are still under the "Unreleased" section. Does this mean that we don't have to add a changelog entry for the changes of this PR, since they only apply a change to something yet to be released?

**Comment by gziolo** [comment]
> Looking at the changelog, the recent changes in the Card component are still under the "Unreleased" section. Does this mean that we don't have to add a changelog entry for the changes of this PR, since they only apply a change to something yet to be released?

It all depends if those changes will get published to npm at the same time. Gutenberg 11.1 was already branched and it probably is going to be used as a base for the next npm publishing. If this PR isn't going to be cherry-picked to 11.1 then it might be valuable to include an individual changelog entry, otherwise you can probably skip it.

**Comment by ciampo** [comment]
> It all depends if those changes will get published to npm at the same time. Gutenberg 11.1 was already branched and it probably is going to be used as a base for the next npm publishing. If this PR isn't going to be cherry-picked to 11.1 then it might be valuable to include an individual changelog entry, otherwise you can probably skip it.

Thank you @gziolo! Just to confirm, I was going to add an entry to the `@wordpress/components` package's changelog — is that correct?

**Comment by gziolo** [comment]
> Just to confirm, I was going to add an entry to the @wordpress/components package's changelog — is that correct?

Yes, correct 👍🏻

**Review by sarayourfriend — APPROVED** [review]

File: packages/components/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 -   `isPrimary`, `isSecondary`, `isTertiary` and `isLink` props in `Button` have been deprecated. Use `variant` instead ([#31713](https://github.com/WordPress/gutenberg/pull/31713)).
 -   `isElevated` prop in `Card` has been deprecated. Use `elevation` instead ([#32566](https://github.com/WordPress/gutenberg/pull/32566)).
+-   `isScrollable` prop in `CardBody` default value changed from `true` to `false` ([#33490](https://github.com/WordPress/gutenberg/pull/33490))
```
Comment (gziolo) [inline packages/components/CHANGELOG.md:13]: Noting that if we publish WordPress packages to npm from Guteberg 11.1 this week we will have to move up this entry and put into its own section.

Comment (ciampo) [reply packages/components/CHANGELOG.md:13]: Is there also going to be a new release of the components package (e.g. 14.2.0)?

Comment (gziolo) [reply packages/components/CHANGELOG.md:13]: Yes, we didn't have it for a while but now that WordPress 5.8 should be out tomorrow, there is no blocker anymore.

**Pull request merged**